pkgs/development/libraries/qt-5/5.15/default.nix

```diff
@@ -141,19 +127,18 @@ let
       qtwebchannel = callPackage ../modules/qtwebchannel.nix {};
       qtwebengine = callPackage ../modules/qtwebengine.nix {};
       qtwebglplugin = callPackage ../modules/qtwebglplugin.nix {};
-      qtwebkit = callPackage ../modules/qtwebkit.nix {};
```

Comment:
Is this the eval error?

Reply:
I don't understand what you mean.

Reply:
ofborg fails to eval everything and until that I can't run nixpkgs-review.